### PR TITLE
(+semver: feature) Add simma device and base functionality

### PR DESCRIPTION
**Pull request by annymsMthd**
Signed-off-by: Joshua Benjamin <joshuabnjmn@gmail.com>

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+APP_NAME := j1708-tester
+BUILD_PATH := ./artifacts
+LINUX_BUILD_PATH = $(BUILD_PATH)/linux/$(APP_NAME)
+LINUX_ARM_BUILD_PATH = $(BUILD_PATH)/arm/$(APP_NAME)
+WINDOWS_BUILD_PATH = $(BUILD_PATH)/windows/$(APP_NAME).exe
+MAC_BUILD_PATH = $(BUILD_PATH)/darwin/$(APP_NAME)
+
+.PHONY: build run package
```
Comment (thzinc) [inline makefile:9]: Is this combination of default targets intended? Seems like an odd order.

File: pkg/common/interpreter.go [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	pidInfo := i.getPidDefinition(int(message[0]))
+
+	sb.WriteString(fmt.Sprintf(";    Requested Parameter %d:\n", message[0]))
+	sb.WriteString(fmt.Sprintf(";      %s\n", pidInfo))
+
+	rd := i.getMidDefinition(int(message[1]))
+	sb.WriteString(fmt.Sprintf(";    Receiver MID: %d - %s\n", message[1], rd))
+}
+
+func (i *J1587Interpreter) getMidDefinition(mid int) string {
```
Comment (thzinc) [inline pkg/common/interpreter.go:47]: Is the intent to eventually build this list up? Is there a well-defined set of MIDs somewhere in the world?

Comment (annymsMthd) [reply pkg/common/interpreter.go:47]: SAEJ1587 is the standard. There should be a list of common MIDs/PIDs there. I'm working on getting it so I can flesh this part out more.

File: makefile (hunk 1 above)
Comment (annymsMthd) [reply makefile:9]: https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Comment (thzinc) [reply makefile:9]: Ahhhhh, I get it now. Thanks

**Pull request merged**